sirbot/pythondev/slack.py

```diff
@@ -470,3 +472,13 @@ def add(self, slack):
                             " `{0}`".format(command.text)
 
         await slack.send(response)
+
+        async def members_joined(self, event, slack, _):
+            if event['channel'] == await slack.channels.get(name='general').id:
+                await slack.channels.get('general', update=True)
+                members = await slack.channels.get(name='general').members
+                if (len(members) % 1000) == 0:
+                    to = await slack.channels.get(name='general').members
+                    message = SlackMessage(to=to)
+                    message.text = 'We have just reached {} members!'.format()
```

Comment:
you forgot to put something in the `format`. I guess it should be the `len(members)`